### PR TITLE
Release Notes v0.36.0RC1

### PR DESCRIPTION
**Pull request by esc**
As title.

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,42 @@
+v0.36.0RC1 (February 03, 2021)
+---------------------------
```
Comment (stuartarchibald) [inline CHANGE_LOG:2]:
```suggestion
v0.36.0 (February 04, 2021)
---------------------------
```

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+The main features in this release is Python 3.9 support. Also, some changes to
```
Comment (stuartarchibald) [inline CHANGE_LOG:4]:
```suggestion
The main feature in this release is Python 3.9 support. Also, some changes to
```

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+@abebeos
+@0x5h4un
+Peter-Jan Gootzen
+John Kirkham
+Graham Markall
+Uwe L. Korn
+Stan Seibert (core dev)
+Valentin Haenel (core dev)
+Siu Kwan Lam (core dev)
+Stuart Archibald (core dev)
```
Comment (stuartarchibald) [inline CHANGE_LOG:38]:
```suggestion
Graham Markall
John Kirkham
Peter-Jan Gootzen
Siu Kwan Lam (core dev)
Stan Seibert (core dev)
Stuart Archibald (core dev)
Uwe L. Korn
Valentin Haenel (core dev)
``@ARF1``
``@abebeos``
``@0x5h4un``
```
suggest alphabetical, and handles at the end with escapes.

File: CHANGE_LOG (hunk 1 above)
Comment (stuartarchibald) [reply CHANGE_LOG:2]: Also, title under line has to match number of chars it's underlining.

File: CHANGE_LOG [hunk 4]
```diff
@@ -1,3 +1,42 @@
...
+* PR #624: Expose pass timings
+* PR #655: Switch encoding to UTF-8 from latin1
+* PR #662: Delete `requirements.txt`
+* PR #666: fix rst syntax in install docs
+* PR #668: Modify cmake options to work with VS2019
+* PR #670: Llvmdev windows 10
+* PR #671: Python 3.9 support
+* PR #673: use build 2 on windows
+* PR #677: Support building with CMake on posix systems
+* PR #682: slight rearrangement of intro
```
Comment (stuartarchibald) [inline CHANGE_LOG:21]: Missing #685

**Review by stuartarchibald — COMMENTED** [review]
Thanks for the patch, few minor things to resolve else looks good.

File: CHANGE_LOG [hunk 5]
```diff
@@ -1,3 +1,42 @@
...
+the build scripts were included: the llvmdev recipe now supports compilation on
+Windows 10 and llvmlite may now be compiled with cmake on POSIX systems.
+Additionally, both the timings from the LLVM passes themselves and the time
+spent within the LLVM thread-safe access lock are now available for diagnostic
+purposes. Lastly, several minor cosmetic changes to the documentation and
+project files are included.
+
+* PR #624: Expose pass timings
+* PR #655: Switch encoding to UTF-8 from latin1
+* PR #662: Delete `requirements.txt`
```
Comment (stuartarchibald) [inline CHANGE_LOG:14]:
```suggestion
* PR #662: Delete ``requirements.txt``
```

**Comment by stuartarchibald** [comment]
Also, you can check the rendering in the "docs/readthedocs.org" build accessible from the list of "checks".

File: CHANGE_LOG [hunk 6]
```diff
@@ -1,3 +1,42 @@
...
+---------------------------
+
+The main features in this release is Python 3.9 support. Also, some changes to
+the build scripts were included: the llvmdev recipe now supports compilation on
+Windows 10 and llvmlite may now be compiled with cmake on POSIX systems.
+Additionally, both the timings from the LLVM passes themselves and the time
+spent within the LLVM thread-safe access lock are now available for diagnostic
+purposes. Lastly, several minor cosmetic changes to the documentation and
+project files are included.
+
```
Comment (stuartarchibald) [inline CHANGE_LOG:11]:
```suggestion

Pull requests:

```

File: CHANGE_LOG (hunk 4 above)
Comment (esc) [reply CHANGE_LOG:23]: added now.

**Comment by esc** [comment]
@stuartarchibald thank you for the review, I have addressed all the comments.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**